src/graphs.jl

```diff
@@ -335,7 +335,7 @@ function LoopSet(mod::Symbol)
         Tuple{Int,Symbol}[],
         Tuple{Int,Int}[],
         Tuple{Int,Float64}[],
-        Int[],Int[],
+        Tuple{Int,NumberType}[],Tuple{Int,Symbol}[],
```

Comment:
I've seen this kind of thing in my own code (e.g., https://github.com/JuliaDebug/LoweredCodeUtils.jl/pull/58), and I'm amazed that `convert(Vector{Tuple{Int,NumberType}}, ::Vector{Int})` doesn't error.

Reply:
Good catch!

Being empty saves Julia the need to convert an `Int` into a `Tuple{Int,Symbol}`, while "converting" the container (sans the contents) is trivial. Would be nice if this were easier to catch.
```julia
julia> convert(Vector{Tuple{Int,Symbol}}, Int[])
Tuple{Int64, Symbol}[]

julia> convert(Vector{Tuple{Int,Symbol}}, Int[3, 4])
ERROR: MethodError: Cannot `convert` an object of type Int64 to an object of type Tuple{Int64, Symbol}
Closest candidates are:
  convert(::Type{T}, ::T) where T<:Tuple at essentials.jl:317
  convert(::Type{T}, ::Tuple{Vararg{Any, N}}) where {N, T<:Tuple} at essentials.jl:318
  convert(::Type{T}, ::CartesianIndex) where T<:Tuple at multidimensional.jl:137
  ...
Stacktrace:
```